genie-agent/src/main/java/com/netflix/genie/agent/execution/statemachine/actions/LaunchJobAction.java

```diff
@@ -77,10 +83,25 @@ protected Events executeStateAction(final ExecutionContext executionContext) {
             throw new RuntimeException("Failed to launch job", e);
         }
 
-        log.info("Job process started (pid: {})", getPid(jobProcess));
+        final long pid = getPid(jobProcess);
+
+        log.info("Job process started (pid: {})", pid);
 
         executionContext.setJobProcess(jobProcess);
 
+        try {
+            this.agentJobService.changeJobStatus(
+                executionContext.getClaimedJobId(),
+                executionContext.getCurrentJobStatus(),
+                JobStatus.RUNNING,
+                "Job running (pid: " + pid + ")"
```

Comment:
Is it of value for users to know the PID or is it just noise?

Reply:
Thought it would be nice not to have to search for it (to do things like monitor resource usage, `kill`, ...